app/src/main/java/com/example/demoappmusic/PlayMusic.kt

```diff
@@ -0,0 +1,112 @@
+package com.example.demoappmusic
+
+import android.app.Service
+import android.content.Intent
+import android.media.MediaPlayer
+import android.os.Binder
+import android.os.Handler
+import android.os.IBinder
+import android.util.Log
+import android.view.View
+import android.view.ViewManager
+import android.widget.*
+import kotlinx.android.synthetic.main.activity_play_song.*
+import java.text.SimpleDateFormat
+
+class PlayMusic : Service() {
+
+    val iBinder = LocalBinder()
+    lateinit var mediaPlayer: MediaPlayer
+    var position: Int = 0
+    var isPlay = false
+
+    inner class LocalBinder : Binder() {
+        fun getService(): PlayMusic {
+            return this@PlayMusic
+        }
+    }
+
+    override fun onBind(intent: Intent): IBinder {
+        return iBinder
+    }
+
+    override fun onDestroy() {
+        mediaPlayer.release()
+        super.onDestroy()
+    }
+
+    fun onSkipNext(
+        text_name: TextView,
+        text_time_current: TextView,
+        text_time_total: TextView,
+        seekBar: SeekBar
+    ) {
+        mediaPlayer.stop()
+        position++
+        if (position > MainActivity.arrSong.size - 1) {
+            position = 0
+        }
+        onPlaySong(text_name, text_time_current, text_time_total, seekBar, position)
+    }
+
+    fun onSkipPrevious(
+        text_name: TextView,
+        text_time_current: TextView,
+        text_time_total: TextView,
+        seekBar: SeekBar
+    ) {
+        mediaPlayer.stop()
+        position--
+        if (position < 0) {
+            position = MainActivity.arrSong.size - 1
+        }
+        onPlaySong(text_name, text_time_current, text_time_total, seekBar, position)
+    }
+
+    fun onPauseSong(): Int {
+        mediaPlayer.pause()
+        return mediaPlayer.currentPosition
+    }
+
+    fun onResumeSong(x: Int) {
```

Comment:
```suggestion
    fun onResumeSong(position: Int) {
```
Biến cần đặt tên rõ nghĩa